python/ccxt/kucoin.py

```diff
@@ -1724,3 +1725,6 @@ def handle_errors(self, code, reason, url, method, headers, body, response, requ
         message = self.safe_string(response, 'msg')
         self.throw_exactly_matched_exception(self.exceptions['exact'], message, message)
         self.throw_exactly_matched_exception(self.exceptions['exact'], errorCode, message)
+        if errorCode != '200000':
```

Comment:
make it the same as bybit, ask if it's different than 200000 right below the errorCode =